packages/replay/test/integration/flush.test.ts

```diff
@@ -286,15 +286,22 @@ describe('Integration | flush', () => {
 
     expect(mockFlush).toHaveBeenCalledTimes(20);
     expect(mockSendReplay).toHaveBeenCalledTimes(1);
+
+    replay.getOptions().minReplayDuration = 0;
   });
 
   it('does not flush if session is too long', async () => {
     replay.timeouts.maxSessionLife = 100_000;
-    jest.setSystemTime(new Date(BASE_TIMESTAMP));
+    jest.setSystemTime(BASE_TIMESTAMP);
 
     sessionStorage.clear();
     clearSession(replay);
     replay['_loadAndCheckSession']();
+    // No-op _loadAndCheckSession to avoid us resetting the session for this test
```

Comment:
actually noticed this test was incorrect before, it just worked because we never reset `minReplayDuration` above 😬 oops...